plugins/operators/model_evaluation/__init__.py

```diff
@@ -821,10 +822,14 @@ def get_scenarios_picker_type(self, ctx, field_name):
 
         # NOTE: may be slow for large datasets
         values = dataset_or_view.count_values(field_name)
+
+        # @todo consider supporting None
+        values.pop(None, None)
+
         return (
             (ShowOptionsMethod.EMPTY, None)
             if not values
-            else (ShowOptionsMethod.CHECKBOX, dict(sorted(values.items())))
+            else (ShowOptionsMethod.CHECKBOX, values)
```

Comment:
Is the sorting happening somewhere else?

Reply:
@imanjra I meant to comment on this in the code.

I believe you included `dict(sorted(values.items()))` so that the checkboxes/lists we produce in the UI will be sorted alphabetically. However, dicts in Python are not **guaranteed** to have a stable ordering in all situations when you iterate over them in general. So if we want stable sorting, we should work with lists of tuples instead.

For now I just removed the sorting since `dict()` could ruin it.

Reply:
update: I added back the sorting because its not harmful and *could* work. But points above still stand.